torch/_inductor/lowering.py

```diff
@@ -3343,8 +3343,8 @@ def upsample_nearestnd(
 
     scales = [i / o for i, o in zip(i_sizes, o_sizes)]
     for i, scale in enumerate(scales_x):
-        if scale:
-            scales[i] = scale
+        if scale is not None:
+            scales[i] = 1.0 / scale
```

Comment:
NIT: It's a bit confusing that `scale` is used in two different ways, maybe the list should be called `inv_scales`?